tools/xml2file/xml2file.py

```diff
@@ -46,6 +48,11 @@ def parse_command_line():
         default=None,
         help='output file encoding.'
     )
+    parser.add_argument(
+        '--clean',
+        action='store_true',
+        help='clean output direcotry (before)'
```

Comment:
__[Misspell]__ "direcotry" is a misspelling of "directory"

<sub>posted by Sider</sub>